### PR TITLE
Add stop notifications

### PR DESCRIPTION
**Pull request by lubej**

### CODE REVIEW

File: src/index.ts [hunk 1]
```diff
@@ -38,7 +39,11 @@ export const monitorCharacteristic = (
       const value = Buffer.from(rawData.buffer, rawData.byteOffset, rawData.byteLength);
       subscriber.next(value);
     })
-  });
+  }).pipe(
+    finalize(() => {
+      BleClient.stopNotifications(deviceId, service, characteristic);
```
Comment (lukaw3d) [inline src/index.ts]: The Rx idiomatic way to do it is to return dispose function inside `new Observable(`

Comment (lubej) [reply src/index.ts]: You probably mean `subscriber.complete()`. But in this case I want to terminate the subscription on device disconnect. Which is out of context in this scope. `finalize` does exactly that, it is a side effect of complete observable.

Comment (lukaw3d) [reply src/index.ts]: I don't understand. I mean:
```ts
  return new Observable<Buffer>(subscriber => {
    BleClient.startNotifications(deviceId, service, characteristic, rawData => {
      const value = Buffer.from(rawData.buffer, rawData.byteOffset, rawData.byteLength);
      subscriber.next(value);
    });
    return () => {
      BleClient.stopNotifications(deviceId, service, characteristic);
    }; 
  });
```

Docs: https://rxjs.dev/guide/observable#disposing-observable-executions

Comment (lubej) [reply src/index.ts]: I see what you mean now. Should work the similar way.

**Review by lukaw3d — APPROVED** [review]

**Pull request merged**